functions/config.bash

```diff
@@ -33,7 +33,14 @@ clean_config_userpw() {
 ## Update java architecture in config file
 ## Valid options: "32-bit" & "64-bit"
 update_config_java() {
-  cond_redirect grep -q '^java_arch' "$CONFIGFILE" && sed -i "s/^java_arch.*/java_arch=$1/" "$CONFIGFILE" || echo "java_arch=$1" >> "$CONFIGFILE"
+  if [ $1 == "64-bit" ] && is_pithreeplus || is_pifour || is_armv7l || is_armv6l; then
```

Comment:
This is giving a shellcheck error:
https://travis-ci.com/github/openhab/openhabian/jobs/326601913#L482